recipe/meta.yaml

```diff
@@ -40,13 +39,16 @@ test:
   requires:
     - pip
     - pytest >=5.0.1
+    - pytest-cov
```

Comment:
```suggestion
```
 `pytest-cov` is used to generate test coverage reports, and is unnecessary for our testing to my knowledge.